flint.ui/src/views/flint/ConfigurationsRothC.vue

```diff
@@ -40,23 +43,35 @@
 
           <RothCTemplate config-paramtype="soil" />
         </div>
+        <div class="mt-4">
+        <Button :btn-size="'auto'" @click="apiRoute_rothc">Run</Button>
```

Comment:
For the Button component, the click event only works when you add `@click.native=`. Can you add the `.native` here? Currently the Run button is not doing anything. 
Also add a little padding bottom / margin bottom below the button please.

Reply:
ok